docs/docs/self-hosting/manage/database/_postgres.mdx

```diff
@@ -35,4 +35,22 @@ Database:
         RootCert:
         Cert:
         Key:
-```
\ No newline at end of file
+```
+
+The admin user is only needed for the installation step but can also be avoided
+by preparing the necessary requirements:
+
+* the zitadel user
+* the zitadel database
+* granting the zitadel user to required permissions
+
+```
+create role zitadel login;
+create database zitadel;
+grant connect, create on database zitadel to zitadel;
```

Comment:
```suggestion
```sql
CREATE ROLE zitadel LOGIN;
CREATE DATABASE zitadel;
GRANT CONNECT, CREATE ON DATABASE zitadel TO zitadel;
```